test/nand.js

```diff
@@ -0,0 +1,15 @@
+import eq from './shared/eq';
+import { nand } from '../src';
+
+describe('nand', function() {
+  it('compared two values with nand', function() {
```

Comment:
Can you pls write test in the style more resembling this ? https://github.com/char0n/ramda-adjunct/blob/4f5068566fd34ccd0d60a2efb15732386cf2a963/test/hasPath.js#L10-L9

Divide the cases + add explicit case for currying.

Thank you